cluster/images/hyperkube/static-pods/etcd.json

```diff
@@ -10,7 +10,7 @@
     "containers": [
       {
         "name": "etcd",
-        "image": "gcr.io/google_containers/etcd-ARCH:2.3.7",
+        "image": "gcr.io/google_containers/etcd-ARCH:2.2.1",
```

Comment:
What was the version released with, and are we certain we can roll back<>forwards from 2.2.1<>2.2.5 .  That what this whole dance was about.